### PR TITLE
Update default control container to v0.5.2

### PR DESCRIPTION
**Pull request by jpculp**
**Issue number:**

N/A

**Description of changes:**

This updates the default control container from v0.5.1 to v0.5.2

**Testing done:**

Using a custom TUF repo, upgraded/downgraded between v1.2.0 and v1.3.0.
In AWS Systems Manager, instance showed agent upgrade/downgrade between 3.0.1209.0 and 3.1.192.0.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by tjkirch — COMMENTED** [review]
LGTM pending testing.

**Review by cbgbt — APPROVED** [review]
Same. Assuming testing looks good, :shipit:

**Review by etungsten — APPROVED** [review]

**Comment by jpculp** [comment]
Renamed 1.2.1 migration to 1.3.0.

I'll re-base and pull out of draft after #1742 merges.

**Comment by jpculp** [comment]
Tested and rebased.

**Review by tjkirch — APPROVED** [review]

**Review by etungsten — APPROVED** [review]

**Pull request merged**
